packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -326,6 +349,35 @@ export class RichTextEditor extends FoundationElement implements ErrorPattern {
         return false;
     }
 
+    public getMentionedHrefs(): string[] {
+        const mentionedHrefs = new Set<string>();
+        this.tiptapEditor.state.doc.descendants(node => {
+            if (node.type.name.startsWith(mentionPluginPrefix)) {
+                mentionedHrefs.add(node.attrs.href as string);
+            }
+        });
+        return Array.from(mentionedHrefs);
+    }
+
+    protected override createConfig(): EditorConfiguration {
+        return new EditorConfiguration(this.mentionElements);
+    }
+
+    private isOnlyMentionInternalsChanged(
```

Comment:
Isn't this more like `isMentionExtensionConfigUnchanged`

At this point it's not about mentionInternals only the configuration objects.

Reply:
Renamed